contrib/certbot/template.py

```diff
@@ -25,11 +42,75 @@
 maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
 license = "Apache-2.0 AND MIT"
 url = "https://github.com/certbot/certbot"
-source = (
-    f"https://github.com/certbot/certbot/archive/refs/tags/v{pkgver}.tar.gz"
-)
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
 sha256 = "7e277bb461cae4071e22641e076d9232ae00ffda05bdb02832cbc1f862afab2d"
 
 
+def post_build(self):
+    for plugin in _plugins:
+        self.do(
+            "python",
+            "-m",
+            "build",
+            "--wheel",
+            "--no-isolation",
+            f"../{plugin}",
+        )
+
+
 def post_install(self):
+    for plugin in _plugins:
+        self.do(
+            "python",
+            "-m",
+            "installer",
+            "--compile-bytecode",
+            "--destdir",
+            self.chroot_destdir,
+            f"../{plugin}/dist/{plugin.replace('-','_')}-{pkgver}-py3-none-any.whl",
+        )
     self.install_license("LICENSE.txt")
+
+
+@subpackage("certbot-plugins")
+def _pluggie(self):
+    self.pkgdesc = "certbot plugins"
+    self.install_if = [f"{pkgname}={pkgver}-r{pkgrel}"]
+    self.options = ["empty"]
+
+    return []
+
+
+def _genmod(pname, pdesc):
+    @subpackage(f"certbot-{pname}")
+    def _plug(self):
+        self.pkgdesc = f"{pdesc} plugin for certbot"
+        self.install_if = [
+            f"{pkgname}={pkgver}-r{pkgrel}",
+            f"certbot-plugins={pkgver}-r{pkgrel}",
+        ]
+        self.depends += [f"{pkgname}={pkgver}-r{pkgrel}"]
+
+        return [
+            f"usr/lib/python*/site-packages/certbot_{pname.replace('-', '_')}*"
+        ]
+
+
+for _plugin, _desc in [
```

Comment:
if these remain split, these descs are weird (DNS should be capitalized, Ovh should be OVH, etc.)

the plugins metapackage pkgdesc is also bad